src/screens/WalletDetailsScreen.tsx

```diff
@@ -42,10 +42,14 @@ interface Props {
 
 export class WalletDetailsScreen extends React.PureComponent<Props> {
   componentDidMount() {
+    this.checkSubscription();
+  }
+
+  checkSubscription = () => {
     const { wallet, email, checkSubscription } = this.props;
     if (!email || !wallet) return;
```

Comment:
Linter could say so many bad things about those three lines here 🗡️ 😄

Reply:
but don't mind me, just a comment. I still think we should improve our linter config though, no rush, but would be nice

Reply:
linter updated in diffrent PR